scripts/update-changelog-from-release.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+
+# Script to update CHANGELOG.md from GitHub release content
+# Usage: ./update-changelog-from-release.sh <tag_name>
+
+TAG_NAME=$1
+
+if [ -z "$TAG_NAME" ]; then
+    echo "Usage: $0 <tag_name>"
+    exit 1
+fi
+
+echo "Updating CHANGELOG.md for release $TAG_NAME..."
+
+# Get release data from GitHub
+RELEASE_DATA=$(gh release view "$TAG_NAME" --json tagName,publishedAt,body,url)
+
+# Extract fields
+TAG=$(echo "$RELEASE_DATA" | jq -r '.tagName')
+DATE=$(echo "$RELEASE_DATA" | jq -r '.publishedAt' | cut -d'T' -f1)
+BODY=$(echo "$RELEASE_DATA" | jq -r '.body')
+URL=$(echo "$RELEASE_DATA" | jq -r '.url')
+
```

Comment:
What do you think about adding a check here to see if the fields did not change and we have data?

```suggestion
if [ -z "$TAG" ] || [ -z "$DATE" ] || [ -z "$URL" ]; then
    echo "Release data incomplete. Please check the release format. Exiting."
    exit 1
fi
```